ui/apps/platform/cypress/integration/vulnmanagement/entitypages.test.js

```diff
@@ -54,151 +54,6 @@ describe('Entities single views', () => {
             });
     });
 
-    // ROX-15888 ROX-15985: skip until decision whether valid to assume high severity violations.
```

Comment:
A fair decision given our experience with this in the past.

Reply:
Yeah, and I noticed that the `skip` tests are for policies, which has been deleted from Vulnerability Management.